docs/gettingstarted/k8s-microk8s.md

```diff
@@ -55,17 +55,26 @@ microk8s helm3 repo update
 ```
 
 # Create a config file
+Dependent on whether you want to store HEC token as a kubernetes secret create `values.yaml` file. 
+If you wish to provide HEC token value in plaintext configure it as in example below:
 
 ```yaml
 --8<---- "docs/resources/k8s/values_basic.yaml"
 ```
-
+Otherwise, if you want to provide HEC token as a kubernetes secret do not provide `hec_token` field  in `values.yaml`:
+```yaml
+--8<---- "docs/resources/k8s/values_basic_no_token.yaml"
+```
```

Comment:
The HEC token can be configured either as a plane text or as a secret.

As Plaintext Configuration:

--8<---- "docs/resources/k8s/values_basic.yaml"

As Secret Configuration:

--8<---- "docs/resources/k8s/values_basic_no_token.yaml"

Reply:
fixed